packages/database/src/migrations/20210610111044-AddReportForMosquitoSpecieesMapOverlay-modifies-data.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+import { generateId, insertObject } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const permissionGroupNameToId = async (db, name) => {
+  const record = await db.runSql(`SELECT id FROM permission_group WHERE name = '${name}'`);
+  return record.rows[0] && record.rows[0].id;
+};
+
+const REPORT = {
+  id: generateId(),
+  code: 'PG_Strive_Mosquito_Species_MapOverlay',
+  config: {
+    fetch: {
+      aggregations: [
+        {
+          type: 'RAW',
+          config: {
+            dataSourceEntityType: 'facility',
+          },
+        },
+      ],
+      dataElements: ['STRVEC_AE-AT13'],
+    },
+    transform: [
+      {
+        transform: 'select',
+        "'numerator'": '1',
+        "'organisationUnitCode'": '$row.organisationUnit',
+        "'name'": '$row.value',
+      },
+      {
+        transform: 'select',
+        "'denominator'":
+          'sum($where(f($otherRow) = equalText($otherRow.organisationUnitCode, $row.organisationUnitCode)).numerator)',
+        '...': ['numerator', 'organisationUnitCode', 'name'],
+      },
+      {
+        transform: 'aggregate',
+        numerator: 'sum',
+        organisationUnitCode: 'group',
+        name: 'group',
+        denominator: 'first',
+      },
+      {
+        transform: 'select',
+        '$row.name': 'fractionAndPercentage($row.numerator, $row.denominator)',
+        "'value'": "'EXISTED'", // Just a tag to indicator we have data
```

Comment:
I think the need for this is something that ideally in the future we could resolve with a more generalised `measureConfig` logic, but for now this hack will have to do 👍 

Perhaps a slight rename suggestion:
```suggestion
        "'value'": "'exists'", // Just a tag to indicator we have data
```